app/jobs/statements/mark_as_paid_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Statements
+  class MarkAsPaidJob < ApplicationJob
+    discard_on StateMachines::InvalidTransition do |_job, exception|
```

Comment:
having a think about this, I've seen it fail in ECF for invalid data that didn't have anything to do with invalid transitions 🤔 I can't remember what it was exactly, but we had to go in and fix the data, I know we said standard error is usually a bad idea but if we're capturing it in Sentry maybe its ok?

Reply:
Cool, will change it :+1: then